drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -107,9 +107,9 @@
 #define APB4_FREQ	((AHB_FREQ)/(CONFIG_CLOCK_STM32_D3PPRE))
 
 /* Datasheet maximum frequency definitions */
-#define SYSCLK_FREQ_MAX	480000000UL
-#define AHB_FREQ_MAX	240000000UL
-#define APBx_FREQ_MAX	120000000UL
+#define SYSCLK_FREQ_MAX	550000000UL
+#define AHB_FREQ_MAX	275000000UL
+#define APBx_FREQ_MAX	137500000UL
 
```

Comment:
You may actually want to run your board at slower clocks at first and then crank it up. 

I know open OCD had some trouble booting my board when I messed up with the clock timings when I coded the H7 clock driver.

Reply:
I'd also like to add that MAX frequencies must depend on the SOC variant, we don't want to allow 500MHz on H743ZI for example. At some point, the H7 clock driver will need some branching for those defines.

Reply:
> You may actually want to run your board at slower clocks at first and then crank it up.
>

I would like to run it on 550MHz, since it is mostly the reason I am introducing this board. However there are different major changes in HW compared to h743 for example, but the clock speed was most interesting for me first. 

> I know open OCD had some trouble booting my board when I messed up with the clock timings when I coded the H7 clock driver.
> 
> EDIT: And also if your clock somewhat went wrong, remember to unplug and replug you board 😄

I am flashing it once with holding reset button on the board, then OpenOCD fail. Then I can flash it as usual many times.

Reply:
> I'd also like to add that MAX frequencies must depend on the SOC variant, we don't want to allow 500MHz on H743ZI for example. At some point, the H7 clock driver will need some branching for those defines.

Agree with that. I will add it within new push.

Reply:
@erwango Is Rev. Y of h7 SoC with 400MHz Sysclk discontinued by ST?

Reply:
So as I understand your board is running fine once you have it flashed for the first time with the Reset technique ?

Just some supposition here, but could it be due to the board using ST-LINK V3E and not V2-1 ? There could be some "Connect Under Reset" option just like in ST-LINK utility that need to be enabled.

Reply:
> So as I understand your board is running fine once you have it flashed for the first time with the Reset technique ?
> 

Yes. But it holds currently only for my `nucleo_h745zi_q` board. This one (`nucleo_h723zg`) I can not currently bring to run with OpenOCD.

> Just some supposition here, but could it be due to the board using ST-LINK V3E and not V2-1 ? There could be some "Connect Under Reset" option just like in ST-LINK utility that need to be enabled.

Testing it currently.

Reply:
@lochej The bug with OpenOCD was in my config (see #29688).

Reply:
Good, so that only was some **Connect Under Reset** thing 👍 Nice you got it sorted out !